interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Save Reports
+author: Mary Sammal
+---
+# Save Reports
+Create a report's _backup copy_ to guarantee that you can revert the report to its original state further.
+
+## Use the Ribbon's Home Tab
+
+* Click **Save** and the **Save as** to save your report's copy.
+	
+	![eurd-win-ribbon-save-report-as](../../../images/eurd-win-ribbon-save-report-as.png)
+	
+	Specify a folder and file name in the invoked standard **Save** dialog.
+
+* Click **Save** or press CTRL+S to save the report's current layout in the application's current folder.
+	
+	![eurd-win-ribbon-save-report](../../../images/eurd-win-ribbon-save-report.png)
+
+
+
+
+## Use the Ribbon Application Menu
+
+Click the application button. The invoked application menu provides the **Save** and **Save as** commands. These commands allow you to save the report's copy and the current layout, correspondingly.
```

Comment:
"the application button"? Which application button?

Reply:
"These commands allow you to save the report's copy and the current layout, correspondingly." One to save a copy, and the other to save the current layout?

Reply:
The application button is illustrated in the image below.

I reread. 
"The invoked application menu provides the **Save** and **Save as** commands. These commands allow you to save the current layout and the report's copy, correspondingly."
One to save the current layout and the other to save a copy. Ok?
Is the following better:
The invoked application menu provides the **Save** command to save the report's current layout, and the **Save as** command to save the report's copy.

Reply:
In the application menu, use the Save command to save the report's current layout, and the Save as command to save the report's copy.
OR
This menu provides the Save command to save the report's current layout, and the Save as command to save the report's copy.